### PR TITLE
chore: add issue template

### PR DESCRIPTION
**Pull request by timstackblock**
Add issue templates for release

### CODE REVIEW

**Comment by timstackblock** [comment]
@agraebe @aulneau issue templates for the deploy are added above please let me know if you want anything else.

**Comment by aulneau** [comment]
@timstackblock could you reword your commit to something like `chore: add issue template`? we follow https://www.conventionalcommits.org/en/v1.0.0/#summary and one of our checks lints our commit mesages. Thanks!

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (agraebe) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: should we add 'bug' or a default importance level?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
```
Comment (agraebe) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: Would it make sense to start with: "Bug: "?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+labels: ''
+assignees: ''
```
Comment (agraebe) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: Would it make sense to assign this to someone by default? Either to triage or directly review it? What could be the best approach here?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```
Comment (agraebe) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]: Maybe put the Transaction ID, timestamp etc at the top?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
```
Comment (agraebe) [inline .github/ISSUE_TEMPLATE/feature_request.md:6]: I'd love to be assigned to feature requests to keep track

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (agraebe) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: Title/label questions as above: would default here make sense?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+assignees: ''
+
+---
+
```
Comment (agraebe) [inline .github/ISSUE_TEMPLATE/feature_request.md:9]: Could we also ask them here is they had a chance to review existing feature requests (link to issues with a feature request tag) and rather upvote/comment as opposed to creating a similar request?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
```
Comment (agraebe) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: Would it make sense to ask for console.logs?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (timstackblock) [reply .github/ISSUE_TEMPLATE/bug_report.md:5]: I will add the priority once I review the issue we have P1, P2, P3 & P4 labels created in this repo.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (timstackblock) [reply .github/ISSUE_TEMPLATE/bug_report.md:6]: Sure I can assign myself to every bug for review

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 5 above)
Comment (timstackblock) [reply .github/ISSUE_TEMPLATE/feature_request.md:6]: I think the label is "enhancement" in this repo and sure we can add that

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 8 above)
Comment (timstackblock) [reply .github/ISSUE_TEMPLATE/bug_report.md:23]: Sure we can that.

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 6 above)
Comment (timstackblock) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: Labels vary depending on severity and title is unique to the bug lets not force a default here.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 4 above)
Comment (timstackblock) [reply .github/ISSUE_TEMPLATE/bug_report.md:11]: Sure no problem

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 6 above)
Comment (timstackblock) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: I think the label is "enhancement" in this repo and sure we can add that

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 7 above)
Comment (timstackblock) [reply .github/ISSUE_TEMPLATE/feature_request.md:9]: Yes I will link them to the the enhancement label in this repo.

**Comment by aulneau** [comment]
@timstackblock can you please rebase this branch and squash the commits, and follow the format of https://www.conventionalcommits.org/en/v1.0.0/#summary? Then we can merge this in. Thanks!!

**Comment by agraebe** [comment]
@aulneau this should be done now

**Pull request merged**